clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/security/DcosCredentialMap.groovy

```diff
@@ -0,0 +1,19 @@
+package com.netflix.spinnaker.clouddriver.dcos.security
+
+class DcosCredentialMap {
```

Comment:
I mean, it is being used like one which is what matters here, not what it is using underneath, that said as you bring up below, we can just use a map underneath anyway.